projects/1.16/assets/forgestory/forgestory/lang/zh_cn.json

```
@@ -0,0 +1,11 @@
+{
+    "error" : "§c",
+    "message.forgestory.storyunselected" : "[§6ForgeStory§r] 欢迎！这个世界包含了 §6ForgeStory§r 模组，所以若要开始游玩/创造你的故事，请使用命令 §c/fs set_story @p 'story_id'§r 来选择它。",
```

```suggestion
    "message.forgestory.storyunselected" : "[§6ForgeStory§r] 欢迎！这个世界安装有§6ForgeStory§r模组，若要开始游玩/创造你的故事，请使用命令§c/fs set_story @p 'story_id'§r来选择它。",
```
方括号这个就不用删了吧（）
毕竟算是分隔作用